pkg/webhooks/errors_test.go

```diff
@@ -19,25 +19,28 @@ import (
 	"testing"
 
 	"emperror.dev/errors"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
**Recommended**: Here package names/aliases should also follow the [Golang package naming convention](https://go.dev/blog/package-names) of lowercasing instead of camelCasing. Similarly to `metav1` or `banzaicloudv1alpha1`.
I know we break this rule a thousand times around the code, but at least I want to prevent increasing the count of those.

Reply:
It ended up like that because I wanted to standardize that name inside the package and I saw that [kafkacluster_validator.go](https://github.com/banzaicloud/koperator/blob/master/pkg/webhooks/kafkacluster_validator.go#L23) and [kafkatopic_validator.go](https://github.com/banzaicloud/koperator/blob/master/pkg/webhooks/kafkatopic_validator.go#L23) use it camel-cased.

I'll change this instance back.
Would you like me to update the other files in the package too during this PR ?  (to have `apierrors`, not `apiErrors`)

Reply:
> Would you like me to update the other files in the package too during this PR ? (to have apierrors, not apiErrors)

+1 It'd be great so we have a cleaner code base

Reply:
Done now in commit 8e3fa9a261b6ba43006fd6f49052f9623f605177